btree/btree.c

```diff
@@ -2933,6 +2969,25 @@ static void level_cleanup(struct m0_btree_oimpl *oi, struct m0_be_tx *tx)
 	m0_free(oi->i_level);
 }
 
+/**
+ * Adds unique node descriptor address to m0_btree_oimpl::i_callback.
+ */
+static void btree_callback_add(void **arr, void *addr)
```

Comment:
This function needs to accept three parameters:
1) Tx in which to storre the callback information.
2) Callback function pointer.
3) Callback data to be passed to the callback function on transaction commit complete.

Reply:
the callback information will be stored in the tx capture based on i_callback.